CHANGELOG.md

```diff
@@ -0,0 +1,8 @@
+# Changelog
+
+The changelog lists most feature changes between each release. The list is automatically created
+based on merged pull requests. Search GitHub issues and pull requests for smaller issues.
```

Comment:
What does this mean? That the changeling won't include all PRs?

Reply:
I expect that there'll be some non-functional changes (i.e. updateing some third party dependencies, changing CHANGELOG entries or a like, that we want to ignore in the CHANGELOG history).